pkg/scaffold/v2/makefile.go

```diff
@@ -50,12 +50,8 @@ IMG ?= {{ .Image }}
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:trivialVersions=true"
 
-# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
-ifeq (,$(shell go env GOBIN))
-GOBIN=$(shell go env GOPATH)/bin
-else
-GOBIN=$(shell go env GOBIN)
-endif
+# Define how controller-gen is executed
+CONTROLLER_GEN ?= go run sigs.k8s.io/controller-tools/cmd/controller-gen
```

Comment:
Can we tag a version here? Would this work?

```suggestion
CONTROLLER_GEN ?= go run sigs.k8s.io/controller-tools/cmd/controller-gen@{{.ControllerToolsVersion}}
```

Reply:
Unfortunately not, I played around a bit with https://github.com/dionysius/go-run-version

Reply:
My question here was, can we tag a version for `go run`? Went to your repo but I see no results there. Is is possible?

I still don't see why the kubebuilder binary will have this dependency outside of the `testdata` packages which are actually other modules so theorically it shouldn't matter. Kubebuilder will just have a string containing the version, but why does that add a dependency? The makefile from the generated code does install and call controller-gen, so the dependency there is clear.

Reply:
to keep it short: correct, that's what I wanted to say, not possible to use a tag/version with `go run` with my tests as well

Reply:
You can mark this conversation as resolved to keep the PR clean.